eval_test.go

```diff
@@ -92,7 +92,7 @@ func TestEvalExpr(t *testing.T) {
 }
 
 func TestExecFile(t *testing.T) {
-	testdata := skylarktest.DataFile("skylark", ".")
+	testdata := skylarktest.DataFile("", ".")
```

Comment:
Please don't remove the "skylark" component of the package name each call.  Just because something is common to every call site does not mean it is the callee's responsibility to do it.

Reply:
My reasoning is not that it is common.
The reasoning is that these arguments are a lie (see `DataFile` signature). It is **not** the package name as the callee suggests but a subpackage name at best. Overall `DataFile` builds a special constructed directory structure, which exclusively works with `go build`.
It does not work with Bazel and might not work for Buck, Gradle or SCons.

Reply:
I see.  I was tempted to suggest that the Bazel DataFile concatenate "com_github_google_"+subdir, but what you have seems fine.